packages/lesswrong/lib/collections/dialogueChecks/collection.ts

```diff
@@ -0,0 +1,65 @@
+import {ensureIndex} from "../../collectionIndexUtils";
+import {addUniversalFields, getDefaultMutations, getDefaultResolvers, schemaDefaultValue} from "../../collectionUtils";
+import {MutationOptions} from "../../vulcan-core/default_mutations";
+import {createCollection} from "../../vulcan-lib";
+import {userOwns} from "../../vulcan-users/permissions";
+//import schema from "./schema";
+
+const schema: SchemaType<DbDialogueCheck> = {
+  // permissions enforced via collection-level checkAccess
+  userId: {
+    type: String,
+    nullable: false,
+    canRead: ['members'], 
+    canCreate: ['members'],
+  },
+  targetUserId: {
+    type: String,
+    nullable: false,
+    canRead: ['members'],
+    canCreate: ['members'],
+  },
+  checked: {
+    type: Boolean,
+    nullable: false,
```

Comment:
This doesn't need an explicit `nullable: false`, since it's provided by default from `schemaDefaultValue`

Reply:
(or rather this will be true after nullability PRs are merged)

Reply:
@darkruby501 when do you expect nullability PRs to be merged?